.gitignore

```diff
@@ -22,3 +22,4 @@ publish/src
 publish/package
 demo*/report/report.html
 demo*/report/stats.json
+/.project
```

Comment:
Where does this folder come from?

Reply:
It's a project file used by some GitHub frontends. It was added automatically from the one I use on Linux.